src/store/socket.js

```diff
@@ -172,6 +172,10 @@ class LiveSession {
         if (!this._isSpectator) return;
         this._store.commit("toggleNight", params);
         break;
+      case "recordVoteHistory":
+        if (!this._isSpectator) return;
+        this._store.commit("session/toggleRecordVoteHistory", params);
```

Comment:
I'd suggest clearing the vote history here, too. When the ST disallows vote histories for players, they shouldn't be able to access old ones either, IMO

Reply:
I guess you're putting it specifically here to avoid the ST clearing their own history when they're not expecting it. Let me know if I misunderstood this...

Reply:
With this added, maybe it makes sense for clearVoteHistory to become just a local operation? 
The ST doesn't seem to need it anymore.